hack/istio/install-istio-via-istioctl.sh

```diff
@@ -330,13 +341,35 @@ if [ "${DELETE_ISTIO}" != "true" ]; then
   # If OpenShift, we need to do some additional things - see:
   #   https://istio.io/latest/docs/setup/platform-setup/openshift/
   echo Creating the control plane namespace: ${NAMESPACE}
-  if [[ "${CLIENT_EXE}" = *"oc" ]]; then
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
     if ! ${CLIENT_EXE} get namespace ${NAMESPACE}; then
       ${CLIENT_EXE} new-project ${NAMESPACE}
     fi
 
-    echo Performing additional commands for OpenShift
-    ${CLIENT_EXE} adm policy add-scc-to-group anyuid system:serviceaccounts:${NAMESPACE}
+    echo "Creating SCC for OpenShift"
+    cat <<SCC | ${CLIENT_EXE} apply -f -
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: istio-openshift-scc
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+fsGroup:
+  type: RunAsAny
+allowPrivilegedContainer: true
+seccompProfiles:
+- '*'
+priority: 9
+groups:
+- "system:serviceaccount:${NAMESPACE}"
```

Comment:
This means that `allowPrivilegedContainer: true`  will be applied to all SAs in istio-system. Shouldn't we add that just for prometheus and grafana SAs? Other SA should follow OCP platform setup specified in Istio doc - `adm policy add-scc-to-group anyuid system:serviceaccounts:${NAMESPACE}`

Reply:
https://github.com/kiali/kiali/pull/6640